go/integTest/test_utils.go

```diff
@@ -3,10 +3,17 @@
 package integTest
 
 import (
+	"context"
 	"strconv"
 	"strings"
+	"time"
+
+	"github.com/stretchr/testify/assert"
 )
 
+// General function type
+type fn func(context.Context)
```

Comment:
The function name should be a little more descriptive like `contextfn` or `fnCtx`. Anything that indicates this function is used where the context object is expected.